app/code/Magento/Sales/Api/Data/OrderItemInterface.php

```diff
@@ -856,7 +856,7 @@ public function getProductId();
     /**
      * Gets the product options for the order item.
      *
-     * @return string[] Array of product options.
+     * @return array Array of product options.
```

Comment:
Not sure. I guess. 
I saw [this](https://github.com/magento/magento2/blob/077584c99ebb8007cad176c3b9a0144a05c259cd/lib/internal/Magento/Framework/Reflection/DataObjectProcessor.php#L110) and automatically assumed that M2 takes out the last 2 chars from `type[]`, leaving just `type`; haven't thought of `@return type1|type2`. I'll try it.